test/Microsoft.AspNetCore.Http.Tests/DefaultHttpContextTests.cs

```diff
@@ -158,7 +158,9 @@ public void UpdateFeatures_ClearsCachedFeatures()
             features.Set<IHttpWebSocketFeature>(new TestHttpWebSocketFeature());
 
             // featurecollection is set. all cached interfaces are null.
-            var context = new DefaultHttpContext(features);
+            var context = new DefaultHttpContext(features, FormOptions.Default);
```

Comment:
Make sure we validate the default ctor as well

Reply:
Done